util_curl.go

```diff
@@ -0,0 +1,77 @@
+package resty
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"net/http/cookiejar"
+
+	"net/url"
+	"strings"
+
+	"github.com/go-resty/resty/v2/shellescape"
+)
+
+func BuildCurlRequest(req *http.Request, httpCookiejar http.CookieJar) (curl string) {
```

Comment:
@ahuigo, please un-export this method to the package.

Reply:
ok